dagger-reflect/reflect/src/test/java/dagger/reflect/ReflectiveMembersInjectorTest.java

```diff
@@ -168,6 +168,31 @@
     assertThat(instance.called).isTrue();
   }
 
+  private static class MethodMultipleDependencies {
+    boolean called;
+
+    @Inject protected void multiple(String one, long two, long two2, @Named("tres") Integer three) {
```

Comment:
What does dagger-compile do with multiple arguments which bind to the same key? Perhaps we should migrate all of these non-failing test cases to be integration tests.

Reply:
Hmm, good question. I don't think I've ever seen it IRL, not even as a mistake.

I'm guessing due to the nature of this library, most of the tests help to be integration tests to make sure it behaves the same as compile + if dagger changes it's easy to notice.

The only drawback is a bit of boilerplate to get the output to be verifiable.